### PR TITLE
Saving geotiff for SA1

### PR DESCRIPTION
**Pull request by xinwei89**
This code aims to download aerial images from NSW six maps based on a specified SA1 code.

It allows you to plot the image and save it as a geotiff.

### CODE REVIEW

**Review by hlydecker — COMMENTED** [review]
Looks good so far; start refactoring some of the functions as you develop them into a seperate .py file that is easier to review.

**Comment by gdmcdonald** [comment]
Get a code review with: flows summarize

**Comment by gdmcdonald** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

**Comment by gdmcdonald** [comment]
sih_bot

**Comment by hlydecker** [comment]
> Discussions with the team before making significant changes could help avoid potential merge conflicts.

lol thanks code review bot.

wow that was one massive wall of text!

**Comment by gdmcdonald** [comment]
Yes that was a ridiculous wall of text 😂😅 Maybe if we edit the prompts it will improve?

You can edit the prompts it uses here:
for each commit diff:
https://github.com/Sydney-Informatics-Hub/github-pr-summary/blob/1022e25e4894f5c0559ba112107d5d301c3c0fff/src/github-pr-summary.rs#L178
for the summary:
https://github.com/Sydney-Informatics-Hub/github-pr-summary/blob/1022e25e4894f5c0559ba112107d5d301c3c0fff/src/github-pr-summary.rs#L208

File: sa1.py
Comment (sahandv) [inline sa1.py:1]: Looks great. Still reading through it. At this time:

* Just have a few questions:

1. Does this library/code need any CLI or executing code? 
2. Is there going to be more of these libraries in this project repo?
3. If both 1 and 2 are yes, then, can we maybe use them as modules?

* Also, can you please generate docstrings for all methods? (starting from line 84)

Thanks 😊

**Review by sahandv — COMMENTED** [review]
This looks great. I have added some comments for the current stage of the code. Please feel free to ignore or change. We can discuss any of the comments, or any other thoughts at any time. 

Thanks 😊

File: .gitignore [hunk 1]
```diff
@@ -158,3 +158,12 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+
+*.csv
+*.tif
+xw*
+*json
+*.txt
```
[the remark this answers is not in the document]
Comment (xinwei89) [reply .gitignore:167]: Thanks, I was just trying to ignore output files used for testing, these should be changed and be more specific later down the track.

[the remark this answers is not in the document]
Comment (sahandv) [reply .gitignore:167]: Oh, thanks for the clarification.

File: sa1.py
Comment (xinwei89) [reply sa1.py:1]: Thanks, I added in the docstrings. The example usage of the classes is shown in `draft.ipynb` cell 5. Let me know how I can improve it!

File: download_raster.py
Comment (hlydecker) [inline download_raster.py:1]: Refactor the repo to have a structure similar to the aerial-segmentation repo:

1. modules in python package
2. scripts in a scripts dir

File: download_raster.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+import os
```
Comment (hlydecker) [inline download_raster.py:2]: Nice clean imports!

**Review by hlydecker — APPROVED** [review]
This looks really good. Let's merge the PR and continue development!

**Pull request merged**